### ISSUE TITLE
Predicting using only 1 specific lambda value

### ISSUE THREAD

**Issue by leocob** [privefl/bigstatsr#164]
I know that the strength of the functions big_spLinReg and big_spLogReg is to automatically perform hyper-parameter optimization for lambda and alpha. However, I was wondering if it's possible to generate a model using only a specific value for lambda and alpha, or alternatively if it's possible to predict on unseen data after CMSA using only a specific value of lambda?

**Comment by privefl** [comment]
No, I don't think it is possible to provide a specific value for lambda.

You can probably provide a value relative to lambda_0 (by setting `nlambda = 2` and `lambda.min.ratio`), but I guess this is not what you want.

**Comment by leocob** [comment]
Ok thank you!

I found out that is possible to do it with the package biglasso, I leave the snippet here since it might be of interest for others:

`predict(fit, X.bm, type="class", lambda=<specific_lambda_value>)`

From the [biglasso pdf manual](https://cran.rstudio.com/web/packages/biglasso/biglasso.pdf):


> **lambda**: Values of the regularization parameter lambda at which predictions are requested. Linear interpolation is used for values of lambda not in the sequence of lambda values in the fitted models.
